src/main/kotlin/me/spradling/gift/core/api/routes/HealthHandler.kt

```diff
@@ -0,0 +1,15 @@
+package me.spradling.gift.core.api.routes
+
+import io.vertx.core.Handler
+import io.vertx.core.json.Json
+import io.vertx.ext.web.RoutingContext
+import me.spradling.gift.core.api.models.responses.health.HealthResponse
+import javax.inject.Singleton
+
+@Singleton
+class HealthHandler: Handler<RoutingContext> {
+
+  override fun handle(context: RoutingContext) {
+    context.response().setStatusCode(200).end(Json.mapper.writeValueAsString(HealthResponse()))
```

Comment:
Constant or hardcode?

Reply:
I'm indifferent about this one. In some ways, the constant is more readable, in some ways `HttpResponseStatus.OK.code()` is nice.